plugins/device/cmdexec/device.go

```diff
@@ -0,0 +1,51 @@
+package cmdexec
```

Comment:
Move only device commands to `plugins/device/deviceutils/utils.go`

    package deviceutils
    func CreatePV
    func CreateVG
    func RemovePV
    func RemoveVG

Move other parts of code back to `transactions.go`. 

 `plugins/device/deviceutils/utils.go` can be moved to `pkg/deviceutils` later if required.

Reply:
Why is there a need to move commands to device utils?, won't it be useful if a separate library is made for commands

Reply:
`cmdexec` is not a good name for library, doesn't give any clue what that cmdexec is. Another problem with current approach is that this library can't be used without having Transaction context. Library should be independent of Transaction/etcd so that it can be reused in other places like Tests.

Reply:
Based on the discussion with you, I grouped the device related functions in separate file here https://github.com/gluster/glusterd2/pull/661/commits/2adafaa97e4c96ba00d20d5fdcdf1e8d2da58f91#diff-388b0ce2a0b41316c76c470557c4cbd1